templates/master.yaml

```diff
@@ -414,7 +411,7 @@ write_files:
       name: service-account-context
     current-context: service-account-context
 
-- path: /etc/kubernetes/ssl/apiserver-crt.pem.enc
+- path: /etc/kubernetes/ssl/apiserver.pem.enc
```

Comment:
@xh3b4sd what's the correct naming scheme here? [This issue](https://github.com/giantswarm/aws-operator/issues/18) says without `-crt`, so just checking to make sure.